### PR TITLE
Add Missing tokenization test [electra]

### PR DESCRIPTION
**Pull request by IMvision12**
# What does this PR do?

Added tokenization test for electra

### CODE REVIEW

**Comment by sgugger** [comment]
Thanks a lot for your PR! @ArthurZucker could you review?

File: tests/models/electra/test_tokenization_electra.py [hunk 1]
```diff
@@ -0,0 +1,335 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Team. All rights reserved.
```
Comment (sgugger) [inline tests/models/electra/test_tokenization_electra.py:2]:
```suggestion
# Copyright 2023 The HuggingFace Team. All rights reserved.
```

**Review by sgugger — APPROVED** [review]
Sorry for the delay, Arthur didn't get to review your PR before leaving for a long vacation. Just one nit and this should be good to merge!

**Comment by IMvision12** [comment]
@sgugger Done!!

**Comment by ArthurZucker** [comment]
Thanks for contributing 🔥

**Pull request merged**